### ISSUE TITLE
As a CIF user, I want to have all the budget-related fields on the budget form, so that related information is grouped together

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-cif#955]
#### Description:

Re-locate the "Total Funding Request" field from the Overview form to the Budgets, Expenses & Payments form and rename "Maximum Funding Amount". 

#### Acceptance Criteria:

Given I am a Project Administrator
When I am creating/editing a project
And I am on the Budgets, expenses and payments form
Then I am able to enter the Maximum Funding Amount related to a project

Given I am a Project Administrator
When I am creating/editing a project
And I am on the Overview form
Then I do not see a Total Funding Request field

#### Development Checklist:

- [ ] update the db using whatever method we determine in this card: https://github.com/bcgov/cas-cif/issues/937
- [ ] update `schemas` and `uiSchema` of both forms (read-only versions too)
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by LindsayMacfarlane** [comment]
Review of ticket (November 8, 2022):
- Max funding amount is currently on the Budgets, Expenses and Payments form
- Would like input from @nanyangpro related to the removal of the total funding request from the project overview page. Is this correct? There isn't really an equivalent field on the budget page (funding is broken down into different categories). Do we need to move this field or is the direction/input from CIF to remove?

**Comment by nanyangpro** [comment]
Thanks for confirming, @LindsayMacfarlane - Amount-wise, "Total Funding Request" is essentially a duplicate of "Maximum Funding Amount". The former isn't as relevant/crucial (a parameter for certain calculations) as the latter to the CIF team once the FA is in place.

However, the "Total Funding Request" is a valid field during Proposal Review & Evaluation Phase and the FA Negotiation Phase. We can keep "Total Funding Request" on the Project Overview page, but let's rename "Max Funding Amount" to "Maximum Funding Amount" on the Budgets page still.

**Comment by LindsayMacfarlane** [comment]
Thanks @nanyangpro - just for my clarity, can this ticket be updated to simply capture the updating of the terminology for Max funding amount?

**Comment by nanyangpro** [comment]
Hi @LindsayMacfarlane, it's covered in #1029. :)

If we decide to keep the "Total Funding Amount" on the Project Overview page, then this ticket can be closed.

**Comment by pbastia** [comment]
I am nott sure if this is still relevant. The funding request is part of the application (which will come in from the external system once this is in place), and the maximum funding amount is part of the funding agreement. I think these are 2 different numbers which might end up being different depending on CIF budget and constraints

**Comment by LindsayMacfarlane** [comment]
Good conversation this morning at Backlog Refinement related to potential impacts the external user portal would have on this approach. Flagging that this ticket wasn't a high priority for the CIF team, so if we have a recommendation, we can take that back to them for consideration.
